umap/settings/docker.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding:utf-8 -*-
+"""
```

Comment:
With docker and kubernetes a lot of configuration can be done with environment variables and not with setting files. If I remember correctly this file and that dependency allows to do that.

Reply:
An example of a configuration in docker can be found in this comment:
https://github.com/umap-project/umap/pull/801#issuecomment-1279765869

Reply:
I think using env for reading some settings make sense, but:

- let's keep it only for the base settings (database URL, redis URL, secret link)
- this should be done in the `settings/base.py`, because maintaining a full settings for docker is not DRY at all and will make the docker image to break often I think

Reply:
I would expect the basic settings to be set with environment variables and the more specific via other means.

Always supplying umap.conf doesn't really simplify things on docker or kubernetes. On kubernetes working with local files isn't recommended and would need extra setup and isn't possible on for instance Google Cloud. One could do it via config maps on kubernetes, but that is more difficult to setup.